test/torch_distributed/test_ddp.py

```diff
@@ -38,6 +39,8 @@ def _ddp_correctness(rank,
   def test_ddp_correctness(self):
     torch_xla.launch(self._ddp_correctness, args=(False, FLAGS.debug))
 
+  # Ref: https://github.com/pytorch/xla/pull/8593
```

Comment:
Did we file separate issue(s) to fix these failures?

Reply:
Not yet. But I will do that right now!